usage/src/com/cloud/usage/UsageSanityChecker.java

```diff
@@ -72,7 +72,9 @@ protected boolean checkItemCountByPstmt(CheckCase checkCase) throws SQLException
         try (PreparedStatement pstmt = conn.prepareStatement(checkCase.sqlTemplate)) {
             if(checkCase.checkId) {
                 pstmt.setInt(1, lastId);
-                pstmt.setInt(2, maxId);
+                if (maxId > 0) {
+                    pstmt.setInt(2, maxId);
```

Comment:
Thanks @rafaelweingartner! 
Absolutely, now I notice there was also a similar [comment](https://github.com/apache/cloudstack/pull/1593/files#r67563826) by @jburwell about this, I should have explained it a little further on this PR description as it can be confusing.